repository-data/application/src/main/resources/hcm-config/hst/hosts.yaml

```diff
@@ -34,6 +34,7 @@ definitions:
                 jcr:primaryType: hst:virtualhost
               jcr:primaryType: hst:virtualhost
             jcr:primaryType: hst:virtualhost
+          hst:scheme: http
```

Comment:
Is it worth adding a comment: # AWS environment will be decommissioned soon, so we won't invest the effort in setting up SSL

Reply:
Agree this would be ideal but unfortunately we can't have comments in YAML at the moment because the YAML Formatter strips them out! 😞